src/main/java/com/ericsson/gerrit/plugins/eiffel/listeners/ChangeMergedEventListener.java

```diff
@@ -64,6 +72,12 @@ protected void prepareAndSendEiffelEvent(Event gerritEvent,
         EiffelEventSender eiffelEventSender = new EiffelEventSender(pluginConfig);
         eiffelEventSender.setEiffelEventType(eiffelEvent.getClass().getSimpleName());
         eiffelEventSender.setEiffelEventMessage(eiffelEvent);
-        eiffelEventSender.send();
+        retryTemplate.execute(new RetryCallback<Void, HttpRequestFailedException>() {
+            @Override
+            public Void doWithRetry(RetryContext context) {
+                eiffelEventSender.send();
+                return null;
+            }
+        });
```

Comment:
This code is duplicated in two classes. Can't we move it to the event sender or the base class

Reply:
It's now called from a shared method in the parent.